orthoinference/src/main/java/org/reactome/orthoinference/SkipTests.java

```diff
@@ -0,0 +1,219 @@
+package org.reactome.orthoinference;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import org.gk.model.ClassAttributeFollowingInstruction;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+public class SkipTests {
+	
+	private static MySQLAdaptor dba;
+//	private static ArrayList<String> skipList = new ArrayList<String>();
+	static HashSet<String> skipList = new HashSet<String>();
+	
+	public static void setAdaptor(MySQLAdaptor dbAdaptor)
+	{
+		dba = dbAdaptor;
+	}
+	
+	// Skiplist was traditionally provided in a file, but since it's currently just 3 instances, I've just hard-coded them here.
+	public static void getSkipList(String skipListFilename) throws NumberFormatException, Exception
```

Comment:
NumberFormatException isn't needed since Exception is more general and will cover it

Reply:
The same is true with other specific exception declarations in methods below

Reply:
Throwing more specific exceptions allows the caller to catch them and handle them, if necessary. The caller might be able to deal with a NumberFormatException, but not the others.

Reply:
These were populated automatically by Eclipse -- it would say something akin to '{Exception name} needed and would do it

Reply:
True, that way is better from a documentation perspective and the other is clearer for what the compiler needs.  Whichever makes more sense and is clearer in the code is fine with me.